__tests__/app.test.js

```diff
@@ -13,6 +14,16 @@ afterAll(() => {
 });
 
 describe("All requests", () => {
+    describe("GET /api", () => {
+        test("200: and return the contents of endpoint.json file", () => {
+            return request(app)
+                .get("/api")
+                .expect(200)
+                .then(({ body }) => {
+                    expect(body).toEqual(descriptionsData);
```

Comment:
I want the test to test the keys and be dynamic. you can't just require in the json file because that will automatically pass

Reply:
Yes, but that's David showed us, that's why. But previously I did dynamic one with read and write file. What shall I do about this ticket. Thanks!